src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1138,5 +1140,28 @@ public Block get(EnchantItemEvent e) {
 				return e.getEnchantBlock();
 			}
 		}, 0);
+		//PlayerFishEvent
+		EventValues.registerEventValue(PlayerFishEvent.class, Entity.class, new Getter<Entity, PlayerFishEvent>() {
+			@Override
+			@Nullable
+			public Entity get(PlayerFishEvent e) {
+				Entity entity = e.getCaught();
+				return entity != null ? entity : null;
+			}
+		}, 0);
+		EventValues.registerEventValue(PlayerFishEvent.class, PlayerFishEvent.State.class, new Getter<PlayerFishEvent.State, PlayerFishEvent>() {
+			@Override
+			@Nullable
+			public PlayerFishEvent.State get(PlayerFishEvent e) {
+				return e.getState();
+			}
+		}, 0);
+		EventValues.registerEventValue(PlayerFishEvent.class, FishHook.class, new Getter<FishHook, PlayerFishEvent>() {
+			@Override
+			@Nullable
+			public FishHook get(PlayerFishEvent e) {
+				return e.getHook();
```

Comment:
Why are you registering event values if you are also creating event value expressions?

Reply:
I thought that you needed both

Reply:
You should try without, and test. 
I may be totally wrong, but I dont think its needed.

Reply:
Okay I'll do that

Reply:
I tested and it seems as if event values are required for event value expressions to work, event value expressions are just like 'aliases' for event values

Reply:
Good to know 👍

Reply:
Another thing, it should automatically check if that exists out of an event of that type, the fact that it doesn't strange

Reply:
Ohh, FishHook is an entity so it thinks there is an entity in various events, okay